### PR TITLE
[ng] ng.io sync mainly of images and script files

### PR DESCRIPTION
**Pull request by chalin**
Contributes to #357.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -44,5 +44,6 @@ pubspec.lock
 a2docs.css
 # !/public/docs/_examples/
 # /public/docs/_examples/node_modules
-# /public/docs/_examples/**/a2docs.css
+/public/docs/_examples/*/dart/web/a2docs.css
+/public/docs/_examples/*/dart/web/styles.css
```
Comment (kwalrath) [inline .gitignore]: Do we really want to ignore styles.css? Oh! Is this so you can have a local copy of the styles.css file without the risk of having different styles.css files?

File: public/docs/_examples/_boilerplate/plnkr.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
```
Comment (kwalrath) [inline public/docs/_examples/_boilerplate/plnkr.json]: Do we really need this file?

File: .gitignore (hunk 1 above)
Comment (chalin) [inline .gitignore]: Right.

File: public/docs/_examples/_boilerplate/plnkr.json (hunk 2 above)
Comment (chalin) [inline public/docs/_examples/_boilerplate/plnkr.json]: Not really, but I've been just sync'ing the entire _boilerplate folder for now. (The file just gets ignored in our build. I can eventually refine the sync filter ... on the next round?)

**Comment by chalin** [comment]
Most of the next images are for guide changes that are forthcoming (or hopefully new guides that we should also be adding soon :).

**Pull request merged**